dessia_common/workflow/core.py

```diff
@@ -673,7 +673,13 @@ def dict_to_arguments(self, dict_: JsonSerializable, method: str):
         raise NotImplementedError(f"Method {method} not in Workflow allowed methods")
 
     def method_dict(self, method_name: str = None, method_jsonschema: Any = None):
-        return {}
+        dict_ = {}
+        for id_, property_ in method_jsonschema['properties'].items():
+            if 'default_value' in property_ :
```

Comment:
This is unfortunately not this trivial, please check the "default_dict" functions in utils/jsonschema.py